vsts_ci/azure-pipelines-e2e.yml

```diff
@@ -13,37 +13,32 @@ stages:
 - stage: E2ETest
   displayName: E2E Tests
   jobs:
-  - job: SetupScript
-    steps:
-    - script: |
-        sudo apt-get update
-        sudo apt-get install jq
-      displayName: 'Install jq'
-    - task: AzureCLI@2
-      inputs:
-        azureSubscription: 'Connection to Pipeline resources for IoT Edge Config'
-        scriptType: 'bash'
-        scriptLocation: 'inlineScript'
-        inlineScript: |
-          az upgrade
-          az extension add --name azure-iot
-          az --version
-          az account set -s $(AzureSubscriptionId)
-      displayName: 'Set Azure resources'
-
   - job: LinuxE2ETests
     pool:
-      vmImage: ubuntu-18.04
+      name: IoT-Devices-E2E
     steps: 
+    - template: continuous-e2e-setup.yml
+    - template: continuous-e2e-cleanup.yml

```

Comment:
>continuous-e2e-cleanup [](http://example.com/codeflow?start=16&length=22)

Maybe call it "continuous-e2e-uninstalledge" as cleanup maybe interpretted as test case or pipeline clean up.